src/main/java/com/glencoesoftware/omero/ms/image/region/PrometheusAuthHandler.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2017 Glencoe Software, Inc. All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.glencoesoftware.omero.ms.image.region;
+
+import java.util.Base64;
+
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.core.http.HttpServerRequest;
+
+class PrometheusAuthHandler implements Handler<RoutingContext> {
```

Comment:
Please use the style of the existing code:

 * Whitespace between statements and curly braces or brackets
 * Opening curly brace on the statement line
 * Whitespace between instance variables